cli/running/running.go

```diff
@@ -581,15 +581,20 @@ func renderInstCtxMembers(instance *InstanceCtx) error {
 	return nil
 }
 
-// CollectInstancesForApps collects instances information per application.
-func CollectInstancesForApps(appList []string, cliOpts *config.CliOpts,
-	ttConfigDir string, integrityCtx integrity.IntegrityCtx) (
-	map[string][]InstanceCtx, error) {
+// GetInstanceEnabledPath returns a path to the directory where all enabled applications are stored.
+func GetInstanceEnabledPath(cliOpts *config.CliOpts, ttConfigDir string) string {
```

Comment:
This API will work, but it is dangerous. It encourages users to create application's path on their own using instances enabled path. This can lead to errors, as there is some tricky logic involved in path generation.
Please add tests to check that the new functionality work with config.yaml, config.yml and instances.enabled = .